lib.sh

```diff
@@ -246,7 +246,7 @@ ask_yes_or_no() {
 
 msg_box() {
     [ -n "$2" ] && local SUBTITLE=" - $2"
-    whiptail --title "$TITLE$SUBTITLE" --msgbox "$1" "$WT_HEIGHT" "$WT_WIDTH"
+    whiptail --title "$TITLE$SUBTITLE" --msgbox "$1" "$WT_HEIGHT" "$WT_WIDTH" 3>&1 1>&2 2>&3
```

Comment:
Why is the last part removed?

Reply:
you mean added? 
Without this the new function won't work, I cannot really explain why.
I can just say that it doesn't introduce unexpected side effects, so I added it to make the new function work

Reply:
Sorry, I'm obviously blind. :D